nuxeo-core/nuxeo-core-event/src/main/java/org/nuxeo/ecm/core/scheduler/ScheduleImpl.java

```diff
@@ -56,6 +56,9 @@ public void setEventId(String eventId) {
     @XNode("cronExpression")
     public String cronExpression;
 
+    @XNode("timezone")
+    public String timeZone;
```

Comment:
Add `@since 10.2` Javadoc on the field and the getter for internal documentation.